lms/urls.py

```diff
@@ -237,6 +237,19 @@
     )
 
 if settings.COURSEWARE_ENABLED:
+    COURSE_URLS = patterns(
```

Comment:
@clintonb i am not sure about adding all the lms urls in here. @chrisndodge what do you have to say about this.